src/app/commons/TextInputForm/TextInputItems.js

```diff
@@ -0,0 +1,16 @@
+import {AiOutlineClose} from "react-icons/ai";
+
+
+const TextInputItems = ({items, removeItem}) => {
+    return items.map((textInputContent) => (
+        <div key={textInputContent.id} className="tag-item">
+            <div className="tag-content">{textInputContent.text}</div>
+            <div className="tag-button"
+                 onClick={() => removeItem(textInputContent)}>
+                <AiOutlineClose size={15}/>
```

Comment:
I think `size={15}` can be moved into the parameters, i.e. line 4: `{items, removeItem, iconSize}`.

Reply:
There is a TODO on the top. It should be refined in another issue. 👍🏻

Reply:
We don't have the demand on changing the icon size, so I will maintain the status. 😄